src/yarn.ts

```diff
@@ -334,11 +336,19 @@ const completionSpec: Fig.Spec = {
 
     const subcommands = packages
       .filter((name) => nodeClis.includes(name))
-      .map((name) => ({
-        name,
-        loadSpec: name,
-        icon: "fig://icon?type=package",
-      }));
+      .map((name) =>
+        name == "@redwoodjs/core"
+          ? {
+              name: ["redwood", "rw"],
+              loadSpec: "redwood",
+              icon: "fig://icon?type=package",
+            }
+          : {
+              name,
+              loadSpec: name,
+              icon: "fig://icon?type=package",
+            }
+      );
```

Comment:
Not sure if this is the best approach, I would prefer something simplier:
```suggestion
      .map((name) => ({
        name: name === "@redwoodjs/core" ? ["redwood", "rw"] : name,
        loadSpec: name,
        icon: "fig://icon?type=package",
      }));
```

Reply:
Agree that is way cleaner, but since it also needs for `loadSpec`, I don't know if is better to repeat the ternary operator or keep it like that ? I let you decide !

Reply:
I think using a ternary operation is both smaller and easier to read, so we can go with this!